en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,88 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signup) or use your GitHub account if you have one (see GitHub instructions below).
```

Comment:
Let's pull the parenthesis out of the sentence
```suggestion
2. Sign up for an account (https://glitch.com/signup) or use your GitHub account if you have one. (See GitHub instructions below)
```
or
```suggestion
2. Sign up for an account (https://glitch.com/signup) or use your GitHub account if you have one. (See GitHub instructions below.)
```

Reply:
Applied second suggestion